store/README.md

```diff
@@ -136,6 +136,18 @@ contains excessive notes and nullifiers, client can make additional filtering of
 * `notes`: `[NoteSyncRecord]` – a list of all notes together with the Merkle paths from `block_header.note_root`.
 * `nullifiers`: `[NullifierUpdate]` – a list of nullifiers created between `block_num + 1` and `block_header.block_num`.
 
+### GetPublicAccountDetails
```

Comment:
You added the documentation to the `store` but not to the `rpc` is that voluntary?

Reply:
This endpoint was removed later.